.github/workflows/scripts/issues/titleCleanup.js

```diff
@@ -0,0 +1,63 @@
+// @ts-check
+const regexes = {
+  // covers 'dataGrid' in all variants, as well as with suffixes (-pro/-premium/etc)
+  dataGrid: /\s?\[\s?(data\s?grid[-\sA-Za-z]*)]/gi,
```

Comment:
```suggestion
  dataGrid: /\s?\[\s?(data\s?grid([-\s](pro|premium))?)]/gi,
```

might want to specifically check for the `-pro/-premium` instead of `-anything`

Reply:
Sometimes it is also DataGridToolbar or -Search or so... better keep it that way tbh 🤔